src/ray/object_manager/object_manager.h

```diff
@@ -0,0 +1,278 @@
+#ifndef RAY_OBJECTMANAGER_H
+#define RAY_OBJECTMANAGER_H
+
+#include <memory>
+#include <cstdint>
+#include <deque>
+#include <map>
+#include <thread>
+#include <algorithm>
+
+// #include <boost/thread.hpp>
+#include <boost/asio.hpp>
+#include <boost/asio/error.hpp>
+#include <boost/bind.hpp>
+
+#include "plasma/plasma.h"
+#include "plasma/events.h"
+#include "plasma/client.h"
+
+#include "ray/id.h"
+#include "ray/status.h"
+
+#include "object_directory.h"
+#include "object_store_client.h"
+#include "format/om_generated.h"
+
+#include "om_client_connection.h"
+
+// #include "ray/common/client_connection.h"
+
+namespace ray {
+
+struct OMConfig {
```

Comment:
`Options` seems to be more widely used.  
https://github.com/google/leveldb/blob/master/doc/index.md#opening-a-database
https://github.com/tensorflow/tensorflow/blob/982549ea3423df4270ff154e5c764beb43d472da/tensorflow/core/grappler/clusters/cluster.h#L123